src/puppetlabs/trapperkeeper/services/status/status_debug_logging.clj

```diff
@@ -0,0 +1,14 @@
+(ns puppetlabs.trapperkeeper.services.status.status-debug-logging
+  (:require [clojure.tools.logging :as log]
+            [schema.utils :refer [validation-error-explain]]
+            [cheshire.core :as json]
+            [puppetlabs.trapperkeeper.services.status.status-core :as status-core]))
+(defn log-status
+  "Log status information at the debug level as json
+
+  Note: This function is in its own namespace so that logback can use the namespace
+  as a way to route these log messages separately from other logging the application
+  might be doing, and so it shouldn't be moved from this namespace"
+  []
+  (let [status (status-core/status-latest-version :debug)]
```

Comment:
Yeah, I don't have a super strong opinion one way or the other but this seems like it will be the common use case, so I'm good with it as-is unless someone feels strongly the other way.